bluesky/run_engine.py

```diff
@@ -305,6 +305,32 @@ def __init__(self, md=None, *, loop=None, preprocessors=None,
 
         self.loop.call_soon(self._check_for_signals)
 
+    def commands(self):
```

Comment:
should it be a `@property`?

Reply:
I think that make sense. Can you also add these to the list of Attributes in the RunEngine's top-level docstring?